apps/teams-test-app/src/components/PagesConfigAPIs.tsx

```diff
@@ -106,6 +106,27 @@ const RegisterOnRemoveHandler = (): React.ReactElement =>
     },
   });
 
+const RegisterOnRemoveHandlerFailure = (): React.ReactElement =>
```

Comment:
Is there no way to fuse this with the above function? I.e if we are sent a failure notifyFailure and if sent a successful response notifySuccess?

Reply:
yes, was trying to avoid actually writing cypress tests :D but I should just write it correctly